src/Logging/DataQueue.h

```diff
@@ -15,11 +15,12 @@
 class DataQueue {
 
     public:
+        enum PublishStatus { Normal, PublishingAtMaxFrequency, DataBufferOverflow };
 
         /**
          * Constructor
```

Comment:
Can you add some descriptions of the params for this function?

Reply:
Sure I can add a comment.  It's not actually a function, just a variable which describes a set of possible states